src/generators/app/templates/basic/src/index.js

```diff
@@ -2,7 +2,7 @@ const PROJECT_NAME = '<%= projectSlug %>';
 const root = document.querySelector(`[data-${PROJECT_NAME}-root]`);
 
 function init() {
-  const App = require('./components/App');
+  import App from './components/App';
```

Comment:
Although this syntax 'works' because of the transpilation step, it's not correct ES Modules usage, because imports should all be top-level and static. Our hot-reloading module-replacement method might benefit from using the new dynamic import syntax (see below), but that means all aunty projects will expect Promises to exist or be polyfilled. Something to think about 🤔

```suggestion
  import('./components/App').then(App => {
    root.appendChild(new App({ projectName: PROJECT_NAME }).el);
  });
```

Reply:
Aaaaaaactually, apparently (because imports are bindings, not assignments, you don't need to re-import like we did with require(). I'll do some tests to confirm, but this might mean we can remove some code 💃

Reply:
If it's any help, hot-reload seems to have been working fine here for at least a week: https://github.com/drzax/rr/blob/master/src/index.js

Reply:
Maybe I'm being pedantic but `ErrorBox` shouldn't find its way into the production build.

Reply:
I wouldn't call that pedantic.

Reply:
@nathanhoad it'll always remain inside a deleted code block